tools/pyclient/README.md

```diff
@@ -0,0 +1,55 @@
+# Installation
+
```

Comment:
installation / usage should be updated to something like ```pip install ....``` . And then we can have a separate section on development